src/common/components/Header.tsx

```diff
@@ -0,0 +1,48 @@
+import type { ReactNode } from "react";
+import styled from "@emotion/styled";
+
+interface HeaderProps {
+  leftElement?: ReactNode;
+  centerElement: ReactNode;
+  rightElement?: ReactNode;
```

Comment:
공통컴포넌트인만큼 타입 범위를 보다 명확히 좁혀줘도 좋을것 같아요!
```suggestion
  leftElement?: ReactElement | string ;
  centerElement:  ReactElement | string;
  rightElement?:  ReactElement | string;
```

Reply:
두 타입 말고 다른 타입 들어올 일 없을 것 같아서 훨 좋아보이네요!!!
[9efe246](https://github.com/Mo-yoy/Moyo-Frontend/pull/19/commits/9efe246856fa4dc34d519229fd9fd2278edaa54f)에서 반영해두었습니당 😃